### PR TITLE
Start common components for ASP.NET Diagnostics Libraries.

### PR DESCRIPTION
**Pull request by iantalarico**
Generalize and pull out the consumer.

### CODE REVIEW

File: apis/Google.Cloud.Diagnostics.Common/Google.Cloud.Diagnostics.Common/project.json [hunk 1]
```diff
@@ -0,0 +1,31 @@
+﻿{
+  "version": "1.0.0-*",
```
Comment (jskeet) [inline apis/Google.Cloud.Diagnostics.Common/Google.Cloud.Diagnostics.Common/project.json]: Not yet! Let's start with 0.1.0, then just before the first release I'll make it 1.0.0-alpha01
We don't want to accidentally go GA :)

File: apis/Google.Cloud.Diagnostics.Common/Google.Cloud.Diagnostics.Common/project.json [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+  "dependencies": {
+    "Google.Api.Gax": "1.0.0-beta05",
+    "NETStandard.Library": "1.6.0"
+  },
+
+
+  "frameworks": {
+    "net45": {},
+    "netstandard1.5": {},
+    "netstandard1.6": {}
```
Comment (jskeet) [inline apis/Google.Cloud.Diagnostics.Common/Google.Cloud.Diagnostics.Common/project.json]: Why are you targeting netstandard1.6 as well?

File: apis/Google.Cloud.Diagnostics.Common/Google.Cloud.Diagnostics.Common/project.json (hunk 1 above)
Comment (iantalarico) [inline apis/Google.Cloud.Diagnostics.Common/Google.Cloud.Diagnostics.Common/project.json]: Hahah done :)

File: apis/Google.Cloud.Diagnostics.Common/Google.Cloud.Diagnostics.Common/project.json (hunk 2 above)
Comment (iantalarico) [inline apis/Google.Cloud.Diagnostics.Common/Google.Cloud.Diagnostics.Common/project.json]: Should have been removed. Fixed.

File: apis/Google.Cloud.Diagnostics.Common/Google.Cloud.Diagnostics.Common/project.json [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+    "requireLicenseAcceptance": false
+  },
+
+  "buildOptions": {
+    "keyFile": "../../GoogleApis.snk"
+  },
+
+  "dependencies": {
+    "Google.Api.Gax": "1.0.0-beta05",
+    "NETStandard.Library": "1.6.0"
```
Comment (jskeet) [inline apis/Google.Cloud.Diagnostics.Common/Google.Cloud.Diagnostics.Common/project.json]: You shouldn't need this dependency, btw - not sure why I missed this before.

**Pull request merged**